internal/target/repository.go

```diff
@@ -269,6 +244,32 @@ func (r *Repository) list(ctx context.Context, resources interface{}, where stri
 	return nil
 }
 
+func (r *Repository) permissionWhereClauses() ([]string, []interface{}) {
```

Comment:
Since this is specific to list permissions, you might want to name this method `listPermissionWhereClauses`

Reply:
Addressed in [aa56f90](https://github.com/hashicorp/boundary/pull/2319/commits/aa56f905816e7b4d6b1eb9e04ea5ba85fc61cb20)